tools/project_manager/autorelease.inc

```diff
@@ -421,28 +421,28 @@ class ReleaseRestrictor
         }
 
         // ---------------------------------------------------------------------
-        // If an author is Anonymous, Unknown, or Various, we should allow 3 books by that 'Author'
+        // If an author is Anonymous, Unknown, or Various, we should allow 4 books by that 'Author'
```

Comment:
Can we reword this and the PM comment so that they are parallel? The PM talks about skips and this one talks about allowing. Having them parallel would make this easier to wrap one's head around.

Consider for here:
```
// Only allow 1 book by the author to be available in this round unless the "author" is 
// one of [Anonymous, Unknown, Various] in which case allow 4 books.
// If they have at least that many, skip their books.
```

And for the PM:
```
// Allow a PM to have 13 books in this round, if they have at least that many skip their books.
```

Reply:
I'm okay with parallelizing, but I'm not sure about the "allow in the round" wording. E.g., it's possible for a PM to have, say, 15 projects in a round. This code doesn't say "that's not allowed!", it just says "I won't release any more for that PM".

I'll suggest a different wording...

Reply:
The new wording is much clearer for me, thank you. Do you want these three commits merged in as-is or are you planning on squashing some of them? I'm happy with either answer.

Reply:
Keep them separate, I'd say.